lib/bolt_spec/plans/mock_executor.rb

```diff
@@ -227,6 +233,13 @@ def publish_event(event)
           end
           @stub_out_message.process(event[:message])
         end
+      elsif event[:type] == :verbose
```

Comment:
This `elsif` doesn't have an `if` before it. Line 235 ends the `if` conditional, so you'll need to delete that and then indent the `elsif` a bit.